### ISSUE TITLE
Questions Regarding Full-Model Fine-Tuning（ Qwen_GR00T）

### ISSUE THREAD

**Issue by vegetabledogzy** [starVLA/starVLA#53]
When performing full fine-tuning, should I only freeze `qwen_vl_interface.model.model.visual`? Or should I freeze nothing at all? Also, should I set the learning rate (lr) to 1e-4 for `qwen_vl_interface` and 1e-5 for the action model? My testing on the Libero dataset achieved a 70% success rate with the entire `qwen_vl_interface` frozen. Now I want to experiment with full fine-tuning.

**Comment by JinhuiYE** [comment]
## Module Freezing Guide

starVLA codebase uses a **regex-based pattern matching** approach to freeze modules.

### How to Freeze Modules:

```python
# 1. Print your model structure to inspect module paths
print(your_model)

# 2. Set module paths to freeze
# Example: freeze the visual module
freeze_module_list = "qwen_vl_interface.model.model.visual"

# Or set to empty string to freeze nothing
# freeze_module_list = ""

# 3. Pass to trainer via command line argument
# --trainer.freeze_modules ${freeze_module_list}
```
